### ISSUE TITLE
Improving over OGame

### ISSUE THREAD

**Issue by Phoscur** [mamen/ugamela#42]
TLDR: This issue shall be more of a collection of ideas with high risk of going offtopic than something that can actually be accomplished.

There might be some minor or major enhancements we could introduce to the game that might make more or less different to the original:

- optional building fleet and defense categorisation(al views) with sorting
- fleet transport helpers (like xorbit has via tampermonkey extension)
- dynamic imperial overview (with fleet control)
- extensibility (tech, buildings, units and resource types)
- more realistic game physics than just using fuel when starting
- extensible composite battle engine

Feel free to add more ideas to this list or rather open a new issue if you think one is actually feasable.
